pkg/engine/loadtargets.go

```diff
@@ -63,8 +73,16 @@ func resolveSpec(i int, target kyvernov1.ResourceSpec, ctx *PolicyContext, logge
 	}, nil
 }
 
-func getTargets(target kyvernov1.ResourceSpec, ctx *PolicyContext, logger logr.Logger) ([]unstructured.Unstructured, error) {
-	var targetObjects []unstructured.Unstructured
+func getTargets(target kyvernov1.ResourceSpec, ctx *PolicyContext) ([]struct {
+	Unstructured      unstructured.Unstructured
+	Subresource       string
+	ParentResourceGVR metav1.GroupVersionResource
+}, error) {
+	var targetObjects []struct {
+		Unstructured      unstructured.Unstructured
+		Subresource       string
+		ParentResourceGVR metav1.GroupVersionResource
+	}
```

Comment:
If this is used in multiple places, maybe define it as a private variables within the package.

Reply:
Done, extracted as a concrete type.